src/commands/newArticle.ts

```diff
@@ -57,18 +57,24 @@ export const newArticleCommand = (context?: AppContext) => {
 
     await vscode.workspace.fs.writeFile(fileUri, text);
 
-    return true;
+    return fileUri;
   };
 
   return () => {
     generator()
-      .then((isCreated) => {
-        if (isCreated) {
+      .then((fileUri) => {
+        if (fileUri) {
           vscode.window.showInformationMessage("記事を作成しました");
+          return fileUri;
         }
       })
       .catch(() => {
         vscode.window.showErrorMessage("記事の作成に失敗しました");
+      })
+      .then((fileUri) => {
+        if (fileUri) {
+          vscode.window.showTextDocument(fileUri);
+        }
       });
```

Comment:
以下のように一つの`.then()`にまとめることもできると思うんですが、どうでしょうか？

```suggestion
      .then((fileUri) => {
        if (fileUri) {
          vscode.window.showInformationMessage("記事を作成しました");
          vscode.window.showTextDocument(fileUri);
        }
      })
      .catch(() => {
        vscode.window.showErrorMessage("記事の作成に失敗しました");
      })
```

Reply:
確かにまとめられそうでした!
まとめて起きます👍